indexer/packages/postgres/src/index.ts

```diff
@@ -17,6 +17,7 @@ export { default as TransferModel } from './models/transfer-model';
 export { default as TradingRewardModel } from './models/trading-reward-model';
 export { default as TradingRewardAggregationModel } from './models/trading-reward-aggregation-model';
 export { default as SubaccountUsernamesModel } from './models/subaccount-usernames-model';
+export { default as LeaderboardPNLModel } from './models/leaderboard-pnl-model';
```

Comment:
we refer to pnl in the codebase as Pnl, lets keep that heuristic and rename to LeaderboardPnlModel, and update accordingly for other objects